Source/UI/CommonUI/View/TableViewCell/SelectionButtonTableViewCell.swift

```diff
@@ -31,7 +31,6 @@ final class SelectionButtonTableViewCell: UITableViewCell {
 }
 
 extension SelectionButtonTableViewCell {
-
```

Comment:
Do we want to always remove this empty line? Personally I find it helpful sometimes and I can see you do too (a lot of your code had this line). Not sure which rule it came from but I'd be happy to disable it